js/apps/admin-ui/src/realm-settings/GeneralTab.tsx

```diff
@@ -78,7 +102,19 @@ export const RealmSettingsGeneralTab = ({
         isHorizontal
         role="manage-realm"
         className="pf-u-mt-lg"
-        onSubmit={handleSubmit(save)}
+        onSubmit={handleSubmit((data) => {
+          save(data);
+          if (
+            UnmanagedAttributePolicy.disabled ===
+            userProfileConfig?.unmanagedAttributePolicy
+          ) {
+            userProfileConfig.unmanagedAttributePolicy = undefined;
+          }
+          adminClient.users.updateProfile({
```

Comment:
I don't like this because the user profile configuration is updated in a separate request. The reason for that is that this new setting does not belong to the realm but to the user profile configuration itself.

We could add a new endpoint to the API used by the console (admin-ui-ext) but I'm not sure.

I don't think adding the user profile configuration representation to the core realm representation is something we want to do.

Reply:
Because we have a specific REST API for User Profile. Introducing user profile concerns to the Realm API is redundant and makes it more complex, IMO.

Reply:
The complexity is there either way, no? This just moves said complexity to the front-end.

Reply:
@jonkoops @mposolda We should have now a single transaction when updating both the realm and user profile. Basically, there is an additional endpoint to `ui-ext` that allows decorating some operations from the core Admin Realm API. In this case, we are updating both realm and user profile settings by leveraging the existing logic from the core APIs.